From f051bc0476952b003c39c6039815d1c1f2513421 Mon Sep 17 00:00:00 2001
From: Edward Liaw <edliaw@google.com>
Date: Tue, 19 Apr 2022 23:13:44 +0000
Subject: [PATCH 15/24] test_execve

Fix compile warning in test_execve.c.

(cherry picked from commit bfb7fbd165f0e7fdd63d3b8806a8b8d31b1b469e)
---
 tools/testing/selftests/capabilities/test_execve.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/capabilities/test_execve.c b/tools/testing/selftests/capabilities/test_execve.c
index df0ef02b40367..08667a0159c71 100644
--- a/tools/testing/selftests/capabilities/test_execve.c
+++ b/tools/testing/selftests/capabilities/test_execve.c
@@ -91,7 +91,7 @@ static bool create_and_enter_ns(uid_t inner_uid)
 	uid_t outer_uid;
 	gid_t outer_gid;
 	int i;
-	bool have_outer_privilege;
+	bool have_outer_privilege = false;
 
 	outer_uid = getuid();
 	outer_gid = getgid();
-- 
2.36.0.550.gb090851708-goog

